gui/src/components/Tooltip.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import Tooltip from '@material-ui/core/Tooltip'
+
+const styles = theme => ({
+  lightTooltip: {
+    background: theme.palette.primary.contrastText,
+    color: theme.palette.text.primary,
+    boxShadow: theme.shadows[24],
+    ...theme.typography.h6
+  }
+})
+
+const StyledTooltips = ({ title, children, classes }) => {
```

Comment:
Let's rename this to be `StyledTooltip` for consistency with its usage as a component:

```js
import StyledTooltip from 'components/Tooltip'
...
<StyledTooltip title={tooltip} placement='left'>
```